clang/test/SemaSYCL/intel-fpga-local.cpp

```diff
@@ -318,11 +345,22 @@ void diagnostics()
   //expected-note@+1 {{did you mean to use 'intel::max_replicates' instead?}}
   [[intelfpga::max_replicates(2)]] unsigned int max_replicates[64];
 
+  //Checking of different argument values
```

Comment:
Do we need to generate AST when we are emitting warnings? In my opinion this  isn't required since we've already checked AST earlier in test and this makes the test more difficult to read. @AaronBallman what is the 'norm' here?

Reply:
We don't have to unless we want to verify the state of the AST after emitting the warning because it's somehow important (which I don't think is the case here).

Reply:
>>Do we need to generate AST when we are emitting warnings? I

No, Removed this. Thanks.